col.go

```diff
@@ -379,7 +380,7 @@ func (c *Column) Grow(w *Window, but int) {
 	}
 
 	// Observation: before I can support lines of arbitrary height, I need to change
-	// Frame to paint partial lines of text.
+	// Frame to paint partial lines of observers.
```

Comment:
You need to revert this change. Please double-check the search & replace reverting.

Reply:
Yes. You *do* need to revert this change.